third_party/terraform/utils/provider.go.erb

```diff
@@ -61,6 +61,11 @@ func Provider() terraform.ResourceProvider {
 					"CLOUDSDK_COMPUTE_ZONE",
 				}, nil),
 			},
+            "scopes": &schema.Schema{
```

Comment:
Although this will be fixed by codegen, can you indent this so it lines up with the rest of the code?